uwblueprint/ccbc
# added migration file for seriesOrder from string to integer + update input field

## Notion ticket link

[Ticket here](https://www.notion.so/uwblueprintexecs/Book-number-field-5c8bb79df1984fcba8035c33538ecae7)



## Implementation description
* changed frontend `AddStringInput` element to `AddNumberInput` element, as well as useState variables from string to number (base series number is 1) 
* change backend model property `series_order` from string to integer
* changed frontend model property `seriesOrder` from string to number
* updated sorting books by series number so it processes number instead of string 



## Steps to test
1. run migration file by following: [these instructions](https://github.com/uwblueprint/ccbc#running-migrations--seeding-database )

2. check that Book number input can only be numbers
<img width="971" alt="Screen Shot 2022-05-21 at 5 11 09 PM" src="https://user-images.githubusercontent.com/53016529/169669115-b84f3f1d-ec70-4ae4-a182-023f52abcde3.png">

3. add multiple books on create review page: check that if you have multiple books, they are displayed in order of increasing book number

4. check if you can publish review of said books



## What should reviewers focus on?
* for the input fields: **(I set minimum book number to be 1, and maximum to be 1000 -> not sure what these numbers should be)**
* currently, if user doesn't enter anything, the book number (on frontend) is set to 1 


## Checklist
